packages/caravan-health/src/waste.ts

```diff
@@ -102,10 +102,10 @@ export class WasteMetrics extends WalletMetrics {
           The amount by which we exceed our selection target when creating a changeless transaction, 
           mutually exclusive with cost of change. It is extra fees paid if we don't make a change output 
           and instead add the difference to the fees.
-      - Input Amount :
-          Sum of amount for each coin in input of the transaction
-      - Spend Amount :
-          Exact amount wanted to be spent in the transaction.
+      - Scripy Type:
+          The script type of the wallet. The change will always be of the same script type as the wallet.
+      - Config:
```

Comment:
Maybe just include the script type here? elsewhere in caravan we talk about a "wallet config" which includes this info (theoretically it could even be a descriptor). 

The actual config json unfortunately nests `m` and `n` in a `quorum` object. You could use the `MultisigWalletConfig` type that is exported from `@caravan/multisig` but if you didn't want to dig into the nested object, either keeping the related args together in the config arg or flattening all into top level args OR all into the same level into a single option object, e.g. 
```javascript
spendWasteAmount({
  weight, 
  feeRate,
  scriptType, 
  requiredSignerCount,
  totalSignerCount,
})
```
Could be cleaner.